packages/foreman/foreman/foreman.spec

```diff
@@ -210,7 +214,11 @@ BuildRequires: (rubygem(coffee-rails) >= 5.0.0 with rubygem(coffee-rails) < 5.1.
 # start specfile facter BuildRequires
 BuildRequires: rubygem(facter)
 # end specfile facter BuildRequires
-
+%else
+%generate_buildrequires
+# TODO: Generate npm
+%{SOURCE1}
```

Comment:
This syntax is super weird, but I guess it's not something we have influence on.
(For others: you start the `%generate_buildrequires` section(macro? whatever) and then every line that is printed on stdout becomes a `BuildRequires: <the line>`.)

Maybe
```suggestion
%generate_buildrequires
# TODO: Generate npm
./%{SOURCE1}
```
so its more obvious we're executing a script here?

Reply:
Indeed
```
Executing(%generate_buildrequires): /bin/sh -e /var/tmp/rpm-tmp.edESHh
+ umask 022
+ cd /builddir/build/BUILD
+ cd foreman-3.11.0-develop
+ /builddir/build/SOURCES/gen-gem-buildreqs
+ RPM_EC=0
++ jobs -p
+ exit 0
Wrote: /builddir/build/SRPMS/foreman-3.11.0-0.3.develop.el9.buildreqs.nosrc.rpm
```